src/utils.rs

```diff
@@ -210,6 +266,44 @@ mod tests {
             load_wallet(&wallet_path).unwrap();
         })
     }
+
+    #[test]
+    // case: wallet path not exist
```

Comment:
The function name itself should describe the test case, so these comments aren't needed.

```suggestion
```